### ISSUE TITLE
Bug in Notebook log query API

### ISSUE THREAD

**Issue by surya-kant** [Esri/arcgis-python-api#1089]
**datetime import is missing in Notebook log**
datetime import is missing in _**arcgis.gis.nb._logs.py**_. Since query method under LogManager, tries to check the instance of **_start_time_** or **_end_time_** value but could not able to find the datetime. This results in NameError: name 'datetime' is not defined.

**To Reproduce**
```# notebook_server is an instance of NotebookServer```
notebook_server.logs.query(end_time=datetime.datetime.today())
```python
logs = notebook_server.logs.query(end_time=datetime.datetime.today())
```
error:
```python
NameError: name 'datetime' is not defined
```
**Fix**
add import 
```python
from datetime import datetime
```

**Comment by achapkowski** [comment]
This will be resolved in version 1.9.1 of the API.  Thank you for reporting this.

**Comment by HakonD** [comment]
@achapkowski Looks like the documentation is way out here. 

While the function expects and tests for a type of datetime.datetime in start_time and end_time ,the documentation on the other hand says string.

```python
"""
    ==================     ====================================================================
    **Argument**           **Description**
    ------------------     --------------------------------------------------------------------
    start_time             Optional string. The most recent time to query.  Default is now.
                            Time can be specified in milliseconds since UNIX epoch, or as an
                            ArcGIS Server timestamp. For example { "startTime": "2011-08-01T15:17:20,123", ... },
                            { "startTime": 1312237040123, ... }, respectively.
    ------------------     --------------------------------------------------------------------
    end_time               Optional string. The oldest time to include in the result set. You
                            can use this to limit the query to the last n minutes or hours as
                            needed. Default is the beginning of all logging.
"""
```
